db/schema.rb

```diff
@@ -1,5 +1,20 @@
 Sequel.migration do
   change do
+    create_table(:cashila, :ignore_index_errors=>true) do
```

Comment:
These tables are from Cashila addon. I guess, they should not be committed here.